README.md

```diff
@@ -304,5 +304,8 @@ For more details see [LICENSE](https://github.com/Udayraj123/OMRChecker/blob/mas
 
 _Find OMRChecker on_ [**_Product Hunt_**](https://www.producthunt.com/posts/omr-checker/) **|** [**_Reddit_**](https://www.reddit.com/r/computervision/comments/ccbj6f/omrchecker_grade_exams_using_python_and_opencv/) **|** [**Discord**](https://discord.gg/qFv2Vqf)
 
+***
+<h1 align="center">Made with ❤️ by Awesome Contributors</h1>
```

Comment:
hi @Harsh-Kapoorr , how about integrating a dynamic contributors page?
https://levelup.gitconnected.com/displaying-contributors-image-on-readme-files-with-no-pain-f1cbaca963e8

Another one is All contributors Bot.